gcp/modules/couchdb/resources/couchdb-network-policy.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: couchdb-network-policy
+  namespace: gpii
+spec:
+  podSelector:
+    matchLabels:
+      app: couchdb
+  policyTypes:
+  - Ingress
+  - Egress
+  ingress:
+  - from:
+    - ipBlock:
+        cidr: 10.0.0.0/24
+    - podSelector:
+        matchLabels:
+          app: couchdb
+    ports:
+    - protocol: TCP
+      port: 4369
```

Comment:
How about adding a comment to reference the network specs/reports? This would help to emphasize the primacy of those security specs as well as eliminating potential redundancy.

Reply:
+1 - I know it's epmd port in ocuchdb, but not sure why it needs to be open and to what it should be open - other CoudchDB pods?

Reply:
@seg10 I like your suggestion.